tfx/orchestration/kubeflow/kubeflow_dag_runner_test.py

```diff
@@ -36,6 +39,9 @@
 
 from ml_metadata.proto import metadata_store_pb2
 
+@component
```

Comment:
shall we change the decorator as @exit_handler(imported from tfx/v1/orchestration/experimental/__init__.py)

Reply:
I was mostly following the [test in v2](https://github.com/tensorflow/tfx/blob/3288393efc889f9db171d3fd81f9dc818a9fce8a/tfx/orchestration/kubeflow/v2/test_utils.py#L326) which uses a container component. Not opinionated here so I can change if preferred.

Reply:
I think @exit_handler is preferred here, it will perform exactly as @component, it's just a recommended syntax for exit handler component.